docs/integration.rst

```diff
@@ -100,163 +91,118 @@ Airflow connection of type `azure_data_lake` exists. Authorization can be done b
-    Executes an copy command from S3 as CSV with or without headers.
+* :py:mod:`airflow.contrib.operators.hive_to_dynamodb`
+* :py:mod:`airflow.contrib.hooks.aws_dynamodb_hook`
 
+Amazon EMR
+''''''''''
 
+For information, look at API Reference:
 
-AWS DynamoDB
-''''''''''''
+* :py:mod:`airflow.contrib.hooks.emr_hook`
+* :py:mod:`airflow.contrib.operators.emr_add_steps_operator`
+* :py:mod:`airflow.contrib.operators.emr_create_job_flow_operator`
+* :py:mod:`airflow.contrib.operators.emr_terminate_job_flow_operator`
 
-:class:`airflow.contrib.operators.hive_to_dynamodb.HiveToDynamoDBTransferOperator`
-     Moves data from Hive to DynamoDB.
+Amazon RedShift
+'''''''''''''''
 
-:class:`airflow.contrib.hooks.aws_dynamodb_hook.AwsDynamoDBHook`
-    Interface with AWS DynamoDB.
+For information, look at API Reference:
 
+* :py:mod:`airflow.contrib.sensors.aws_redshift_cluster_sensor`
+* :py:mod:`airflow.contrib.hooks.redshift_hook`
+* :py:mod:`airflow.operators.redshift_to_s3_operator`
+* :py:mod:`airflow.operators.s3_to_redshift_operator`
 
-AWS Lambda
-''''''''''
+Amazon SageMaker
+''''''''''''''''
 
-:class:`airflow.contrib.hooks.aws_lambda_hook.AwsLambdaHook`
-    Interface with AWS Lambda.
+For instructions on using Amazon SageMaker in Airflow, please see `the SageMaker Python SDK README`_.
 
+.. _the SageMaker Python SDK README: https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/workflow/README.rst
 
-AWS Kinesis
-'''''''''''
+Additional information is available at API Reference:
 
-:class:`airflow.contrib.hooks.aws_firehose_hook.AwsFirehoseHook`
-    Interface with AWS Kinesis Firehose.
+* :py:mod:`airflow.contrib.hooks.sagemaker_hook`
+* :py:mod:`airflow.contrib.operators.sagemaker_training_operator`
+* :py:mod:`airflow.contrib.operators.sagemaker_tuning_operator`
+* :py:mod:`airflow.contrib.hooks.azure_container_registry_hook`
+* :py:mod:`airflow.contrib.operators.sagemaker_model_operator`
+* :py:mod:`airflow.contrib.operators.sagemaker_transform_operator`
+* :py:mod:`airflow.contrib.operators.sagemaker_endpoint_config_operator`
+* :py:mod:`airflow.contrib.operators.sagemaker_endpoint_operator`
 
+Amazon Simple Storage Service (Amazon S3)
+'''''''''''''''''''''''''''''''''''''''''
 
-Amazon SageMaker
-''''''''''''''''
+For information, look at API Reference:
 
-For more instructions on using Amazon SageMaker in Airflow, please see `the SageMaker Python SDK README`_.
+* :py:mod:`airflow.hooks.S3_hook`
+* :py:mod:`airflow.operators.s3_file_transform_operator`
+* :py:mod:`airflow.contrib.operators.s3_list_operator`
+* :py:mod:`airflow.contrib.operators.s3_to_gcs_operator`
+* :py:class:`airflow.contrib.operators.gcp_transfer_operator.S3ToGoogleCloudStorageTransferOperator`
+* :py:mod:`airflow.operators.s3_to_hive_operator.S3ToHiveTransfer`
 
-.. _the SageMaker Python SDK README: https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/workflow/README.rst
+Amazon Web Service Batch Service
```

Comment:
AWS vs "Amazon Web Service" (an it should be Service**s** I think if we expand it)